deno/payloads/v10/auditLog.ts

```diff
@@ -27,6 +28,12 @@ import type { Snowflake } from '../../globals.ts';
  * https://discord.com/developers/docs/resources/audit-log#audit-log-object-audit-log-structure
  */
 export interface APIAuditLog {
+	/**
+	 * List of application commands found in the audit log
+	 *
+	 * See https://discord.com/developers/docs/interactions/application-commands#application-command-object
+	 */
+	application_commands: APIApplicationCommand[];
```

Comment:
Is this field always present (even for audit log entries that do not affect a command) or can it be undefined too?